src/sentry/feedback/models.py

```diff
@@ -27,4 +27,4 @@ class Meta:
         db_table = "feedback_feedback"
         index_together = [("project_id", "date_added")]
 
-    __repr__ = sane_repr("project_id", "event_id")
+    __repr__ = sane_repr("project_id", "feedback_id")
```

Comment:
didn't catch this in an earlier PR -- we renamed the column from `event_id` to `feedback_id`